src/include/execution/sql/functions/string_functions.h

```diff
@@ -21,6 +21,9 @@ class StringFunctions {
   /** This class cannot be copied or moved. */
   DISALLOW_COPY_AND_MOVE(StringFunctions);
 
+  /** Compute ASCII(str). */
+  static void ASCII(exec::ExecutionContext *ctx, Integer *result, const StringVal &str);
```

Comment:
It seems like all the existing built in functions have parameters of the format `func(Type *result, exec::ExecutionContext *ctx, Inputs... )`, but these PRs switch `result` and `ctx`. Should I change them to conform to the existing pattern?

Reply:
Yes switch to match the new code!

Reply:
Fixed